workers/Dockerfile.base

```diff
@@ -46,9 +46,13 @@ RUN apt-get update -qq && \
 
 COPY workers/requirements.txt .
 
+# Pycurl special options are necessary because GnuTLS drops
+# SSL connections and caused this error:
+# https://stackoverflow.com/questions/41084351/django-celery-sqs-boto-exception-sqserror-sqserror-599-gnutls-handshake
 RUN \
   pip3 --no-cache-dir install pip --upgrade && \
   pip3 --no-cache-dir install setuptools --upgrade && \
   pip3 --no-cache-dir install wheel --upgrade && \
   pip3 --no-cache-dir install -r requirements.txt --upgrade && \
+  pip3 --no-cache-dir install pycurl==7.43.0 --global-option="--with-openssl" && \
```

Comment:
Will we need to change this pycurl version number right here every time pycurl gets updated? I'm guessing we want specifically this version instead of the newest one available?

Reply:
I know this version works. The newest one could also work, but upgrading to the newest one automatically seems somewhat scary on the off chance that something breaks.

Reply:
Ok, so you will manually change it here whenever needed?

Reply:
Yep :)

Reply:
Ok, just making sure :)